### DISCUSSION TITLE
Error while creating a MP4 Movie

### DISCUSSION THREAD

**Discussion by jddorellanao** [pyvista/pyvista#3671] [Q&A] [answered]
Hi community, I'm trying to run [this file](https://github.com/keurfonluu/fteikpy/blob/master/.github/sample.py) that creates [this animation](https://github.com/keurfonluu/fteikpy/blob/master/.github/sample.gif) but while running the code, there is an error when saving the frame
```
File "d:\a13\envs\raytrace\sample.py", line 122, in <module>
    p.write_frame()

To retrieve an image after the render window has been closed, set:

 ``plotter.store_image = True``

before closing the plotter.
```
I've tried to write the `p.store_image = True` at line 78 but got another error 
```
File "d:\a13\envs\raytrace\sample.py", line 89, in <module>
    p.show()
RuntimeError: This plotter is closed and unable to save a screenshot.
```

Any suggestions?

**Comment by banesullivan** [comment, marked as answer]
This is odd. I see you're using `auto_close=False` in the `show()` call. Are you hitting the exit button on the window to return to script execution? Perhaps try hitting the `q` key to close the renderer and go back to the script. This behavior might be different on different OS.

Hope this helps!

Also. AWESOME graphic! Love seeing some geophysical data

**Reply by jddorellanao** [reply]
Thank you! It worked!
